runtime/port/unix/j9sysinfo.c

```diff
@@ -1668,6 +1668,17 @@ j9sysinfo_get_cache_info(struct J9PortLibrary *portLibrary, const J9CacheInfoQue
 			omrcpu_get_cache_line_size(&result);
 #endif
 		}
+#elif defined(LINUX) && defined(_SC_LEVEL1_DCACHE_LINESIZE)
+	if (query->cmd == J9PORT_CACHEINFO_QUERY_LINESIZE
+	    && query->cacheType == J9PORT_CACHEINFO_DCACHE
+	    && query->level == 1) {
```

Comment:
Can you implementation used on x86 be extended to aarch64 (assuming that's the problem you're trying to solve)?

It currently looks up the DCACHE info here and caches it in a port global:
https://github.com/eclipse/openj9/blob/cdb8f937a8f37b78e4f869ae1def70ee29ab327f/runtime/port/unix/j9sysinfo.c#L1335-L1340

In fact, there's already a mostly common place to look this data up:
https://github.com/eclipse/openj9/blob/cdb8f937a8f37b78e4f869ae1def70ee29ab327f/runtime/port/unix/j9sysinfo.c#L1645-L1649

Reply:
I tried `sysconf(_SC_LEVEL1_DCACHE_LINESIZE)` on x86_64 and aarch64, and both seemed to work as exptected.

Reply:
I guess I misunderstood your comment last night.

The implementation for x86 reads `/sys/devices/system/cpu/cpu%d/cache/`.
https://github.com/eclipse/openj9/blob/42ed0b81d8489695c2512c58aebdfad413481059/runtime/port/unix/j9sysinfo.c#L1305

On the other hand, the `cache` directory is unavailable on Ubuntu 18.04 for aarch64 as shown below, and I cannot simply use the x86 implementation:
```
$ ls -l /sys/devices/system/cpu/cpu0/
total 0
-rw-r--r-- 1 root root 4096 Feb 21 01:20 cpu_capacity
lrwxrwxrwx 1 root root    0 Mar 27  2019 cpufreq -> ../cpufreq/policy0
-r-------- 1 root root 4096 Feb 21 01:20 crash_notes
-r-------- 1 root root 4096 Feb 21 01:20 crash_notes_size
drwxr-xr-x 2 root root    0 Feb 21 01:20 hotplug
lrwxrwxrwx 1 root root    0 Feb 21 01:20 of_node -> ../../../../firmware/devicetree/base/cpus/cpu@0
-rw-r--r-- 1 root root 4096 Mar 27  2019 online
drwxr-xr-x 2 root root    0 Feb 21 01:20 power
drwxr-xr-x 3 root root    0 Feb 21 01:20 regs
lrwxrwxrwx 1 root root    0 Jan  1  1970 subsystem -> ../../../../bus/cpu
drwxr-xr-x 2 root root    0 Mar 27  2019 topology
-rw-r--r-- 1 root root 4096 Jan  1  1970 uevent
```

It seems the L1 data cache line size is the only information the VM wants from `j9sysinfo_get_cache_info()`, and `sysconf(_SC_LEVEL1_DCACHE_LINESIZE)` should be a most simple and portable way to get the information.

Reply:
#8711 stops `pltest` to call `j9sysinfo_test_get_levels_and_types()` on aarch64.

Reply:
@DanHeidinga What do you want me to do with this?  Should I limit my change to aarch64 only?

Reply:
@DanHeidinga It is more than a week since I posted the comment above.  What should I do with this PR?

Reply:
Sorry @knn-k, just getting back to this after finishing the JDK14 reviews.

For this PR, please limit this to the aarch64 platform.  As a follow on task, it might be worth looking at modifying the x86 implementation to use sysconf for the dcache as well but I don't want to hold up the arm work for that.